Traceback (most recent call last):
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/kvarada/opt/miniconda3/envs/cpsc330/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from lightgbm.sklearn import LGBMClassifier

X_train = train_df.drop(columns=["Target"])
y_train = train_df["Target"]
X_test = test_df.drop(columns=["Target"])
y_test = test_df["Target"]
model = LGBMClassifier(random_state=123)
model.fit(X_train, y_train)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [7][0m, in [0;36m<cell line: 1>[0;34m()[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mlightgbm[39;00m[38;5;21;01m.[39;00m[38;5;21;01msklearn[39;00m [38;5;28;01mimport[39;00m LGBMClassifier
[1;32m      3[0m X_train [38;5;241m=[39m train_df[38;5;241m.[39mdrop(columns[38;5;241m=[39m[[38;5;124m"[39m[38;5;124mTarget[39m[38;5;124m"[39m])
[1;32m      4[0m y_train [38;5;241m=[39m train_df[[38;5;124m"[39m[38;5;124mTarget[39m[38;5;124m"[39m]

[0;31mModuleNotFoundError[0m: No module named 'lightgbm'
ModuleNotFoundError: No module named 'lightgbm'

